Gemfile.rails-3.0

```diff
@@ -6,6 +6,7 @@ gem 'rails', '~> 3.0.0'
 gem 'sqlite3', platforms: [:ruby]
 gem 'activerecord-jdbcsqlite3-adapter', platforms: [:jruby]
 gem 'activerecord-import'
+gem 'tins', '~> 1.6.0', platforms: [:ruby_19]
```

Comment:
tins is automatically pulled in by `coveralls`. For ruby_19, I had to lock it to 1.6.0 which is the last version that supports Ruby 1.9